src/main/scala/com/cloudera/finance/examples/HistoricalValueAtRiskExample.scala

```diff
@@ -47,8 +47,8 @@ object HistoricalValueAtRiskExample {
     def loadTS(inputDir: String, lower: DateTime, upper: DateTime): TimeSeriesRDD = {
       val histories = YahooParser.yahooFiles(inputDir, sc)
       histories.cache()
-      val start = histories.map(_.index.first).takeOrdered(1).head
-      val end = histories.map(_.index.last).top(1).head
+      val start = histories.map(_.index.first()).takeOrdered(1).head
```

Comment:
They're defined as empty-args methods so should be called that way. Or at least that's the theory. empty-args methods have side effects and no-arg methods don't, or that's the convention.